### ISSUE TITLE
Can I manually change the locale instead of following the system default locale？

### ISSUE THREAD

**Issue by hugetiny** [flutter/flutter#121784]
**Expected results:** 
I need a changeLocale or updateLocale function in flutter localization.
**Actual results:** 
not found

**Comment by danagbemava-nc** [comment]
Yes, you can manually do this. You just need to set the locale property of your `MaterialApp/CupertinoApp` and modify that value whenever you want to change it. 

See the following resources
https://stackoverflow.com/questions/65307961/button-to-change-the-language-flutter
https://localizely.com/blog/flutter-localization-step-by-step/?tab=using-gen-l10n#change-app-language-programmatically-t1

Closing this as this is already possible. 

Thank you
